### PR TITLE
add /supply stats service; replace blockchain comparison

### LINKED ISSUE #483: add supply stats to main tab
Lots of users ask for the details and these direct links aren't ideal. Replace the blockchain comparison (outdated) with both total and circulating supply stats? 

https://explorer.gochain.io/totalSupply

https://explorer.gochain.io/circulatingSupply

### PR DESCRIPTION
**Pull request by jmank88**
Fixes #483

Draft until we have good data to test with.

### CODE REVIEW

File: front/src/app/scenes/home/home.component.html [hunk 1]
```diff
@@ -3,37 +3,23 @@
...
+              </ng-container>
+            </dd>
+            <dt>Circulating</dt>
+            <dd>
+              {{supplyStats.circulating | weiToGO | bigNumber}} GO
+              <ng-container *ngIf="supplyStats.locked > 0">
+                ({{supplyStats.locked | weiToGo | bigNumber}} locked)
+              </ng-container>
+            </dd>
+          </dl>
```
Comment (jmank88) [inline front/src/app/scenes/home/home.component.html:22]: e.g.:

Total
1,125,213,837 GO (134,123.12312 fees burned)

Circulating
1,070,213,843.582844 GO (55,213,837.23423 locked)

File: front/src/app/scenes/home/home.component.html [hunk 2]
```diff
@@ -3,37 +3,23 @@
...
-              <td>13 tps</td>
-              <td>$0.01-$0.50</td>
-            </tr>
-            </tbody>
-          </table>
+          <div class="card-title"><img src="../../../assets/icons/comparison.svg"> SUPPLY STATS</div>
+          <dl>
+            <dt>Total</dt>
+            <dd>
+              {{supplyStats.total | weiToGO | bigNumber}} GO
```
Comment (jmank88) [inline front/src/app/scenes/home/home.component.html:10]: Build is failing on these pipes. Not sure how to make them available from here...

Comment (treeder) [reply front/src/app/scenes/home/home.component.html:10]: @r-gochain do you know how to fix this?

Comment (r-gochain) [reply front/src/app/scenes/home/home.component.html:10]: some react magic should be included AFAIR, I'll check this out

Comment (r-gochain) [reply front/src/app/scenes/home/home.component.html]: @jmank88 @treeder build fixed

Comment (jmank88) [reply front/src/app/scenes/home/home.component.html]: Thanks!

**Review by treeder — APPROVED** [review]

**Pull request merged**